mern-ecommerce/controllers/auth.js

```diff
@@ -62,20 +62,20 @@ exports.requireSignin = expressJwt({
   userProperty: 'auth',
 });
 
-exports.isAuth = (req, res, next) => {
-  let user = req.profile && req.auth && req.profile._id == req.auth._id;
-  if (!user) {
+exports.isAdmin = (req, res, next) => {
+  if (req.profile.role === 0) {
```

Comment:
In the `isAdmin` function, the role check is hardcoded to `0`. This might not be flexible or scalable. Consider defining the admin role in a constants file or environment variable to make it easily configurable.